packages/cozy-client/src/models/applications.js

```diff
@@ -0,0 +1,69 @@
+import get from 'lodash/get'
+const STORE_SLUG = 'store'
+
+/**
+ * Return Store URL where an app/konnector can be installed / updated
```

Comment:
```suggestion
 * Returns the store URL to install/update an app/konnector
```

Reply:
I'm wondering who write this comment first... 🤔 ^^"